hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/SCMCertificateClient.java

```diff
@@ -60,6 +60,11 @@ public SCMCertificateClient(SecurityConfig securityConfig,
     super(securityConfig, LOG, certSerialId, component);
   }
 
+  @Override
+  protected boolean handleExpiration() {
```

Comment:
I would add a comment here or to DefaultCertificateClient to explain that we don't want to reinit SCM certs because they are the CA.

Reply:
done.